### ISSUE TITLE
SimEx-Lite GaussianSourceCalculator test

### ISSUE THREAD

**Issue by JunCEEE** [PaNOSC-ViNYL/SimEx-Lite#10]
@godot11 Could you please help test the `GaussianSourceCalculator` in the [libpyvinyl branch](https://github.com/PaNOSC-ViNYL/SimEx-Lite/tree/libpyvinyl) of SimEx-Lite?

## Quickstart
```
git clone https://github.com/PaNOSC-ViNYL/SimEx-Lite.git
git checkout libpyvinyl
```
And then have a look at `tests/test_GaussianSourceCalculator.py`. The [WPG package](https://github.com/samoylv/WPG) needs to be installed for that.

After testing, could you please let me know:
- What do you think blocked you from using the program?
- What do you think is needed to let you start easier?

**Comment by godot11** [comment]
I installed the module as suggested and ran the two test cases in `tests/test_GaussianSourceCalculator.py`. I also took a look at the produced HDF file itself.

Apart from an unharmful pip dependency error during installation, already fixed in [40ec02a](https://github.com/PaNOSC-ViNYL/SimEx-Lite/commit/40ec02a7f69ee1e5f545c05e844fc7bfcb1d6536), everything related to SimEx-Lite went as intended. The test cases were executed without error, and the calculation itself gave a (seemingly) correct result. 

I realized Carsten made some comments in the test module about how we should expand the test cases. I didn't took this task as I wasn't sure if I'm supposed to, but please tell me if you'd like me to work on this.

**Comment by JunCEEE** [comment]
Hi @godot11, thanks a lot for your testing! It would be really nice if you could work on the test cases. I would suggest you create a branch for that and submit a merge request when you finished. We can create a SimEx-Lite issue about the test cases and add the issue to the [sprint kanban](https://github.com/orgs/PaNOSC-ViNYL/projects/2/views/1).

Please remember to add the form factor task to the kanban as well. I would suggest taking that task as your first priority.
